v-next/hardhat/src/internal/cli/telemetry/sentry/anonymizer.ts

```diff
@@ -0,0 +1,379 @@
+import type { Event, Exception, StackFrame, Stacktrace } from "@sentry/node";
+
+import * as path from "node:path";
+
+import {
```

Comment:
Maybe add a comment explaining that this is run in a subprocess so it's fine to load them here, in the top-level

Reply:
[Done here](https://github.com/NomicFoundation/hardhat/pull/5570/commits/480589a11adfeba18855a8be0679e4241aa9f8d8) with also another comment explanation